kpi/views/v2/asset.py

```diff
@@ -618,6 +620,68 @@ def metadata(self, request):
         metadata = self.get_metadata(queryset)
         return Response(metadata)
 
+    @action(detail=False, methods=['GET'],
+            renderer_classes=[renderers.JSONRenderer],
+            url_path=r'migrate(?:/(?P<task_id>[\d\w\-]+))?')
+    def migrate(self, request, task_id: str = None, **kwargs):
```

Comment:
Just wondering: why in the `AssetViewSet`? It feels crowded in here :blush: 
![giphy](https://user-images.githubusercontent.com/2085013/107726044-c088c000-6cb5-11eb-9565-1b7f16b69a4d.gif)

Reply:
Because it's related to assets. 
BTW, I really love the GIF. 
Any suggestions to move it somewhere else?

Reply:
I moved it under `/api/v2/users/<username>/`

Reply:
Thanks